### PR TITLE
Animation updates

### PR DESCRIPTION
**Pull request by Posandu**
You can merge now @blueedgetechno

### CODE REVIEW

**Comment by blueedgetechno** [comment]
remove the do-not-merge tag, whenever you want me to review and merge

**Comment by Posandu** [comment]
### Done.. Now review & merge

**Comment by yashash1511** [comment]
image

you are replacing ( display: none with opacity: 0 ) which may cause overflow on some android devices (especially tablets)

File: src/components/start/sidepane.scss [hunk 1]
```diff
@@ -8,14 +8,13 @@
...
   overflow: hidden;
-  transition: all ease-in-out 200ms;
+  transition: all ease-in 0.2s;
   z-index: 9999;
   transform: translateX(0);
 
   &[data-hide="true"] {
     bottom: 0.9rem;
     transform: translateX(110%);
-    animation: overflow-hider 0s ease-in-out 0.2s forwards;
```
Comment (yashash1511) [inline src/components/start/sidepane.scss:18]: don't remove this line and the overflow will be fixed :)

**Review by yashash1511 — CHANGES_REQUESTED** [review]

File: src/components/start/sidepane.scss (hunk 1 above)
Comment (blueedgetechno) [reply src/components/start/sidepane.scss:18]: yes, @Posandu take care of this.

**Comment by yashash1511** [comment]
image

even the right click menu is showing overflow, I am not sure about this but it could be due to replacing (display: none with opacity: 0 )

File: src/components/start/sidepane.scss (hunk 1 above)
Comment (Posandu) [reply src/components/start/sidepane.scss]: Done

**Comment by yashash1511** [comment]
lgtm now :)

**Review by yashash1511 — COMMENTED** [review]
lgtm now  👍

though @Posandu  i would suggest merging the commits into fewer commits so that it doesn't spam the commit history :)

**Comment by blueedgetechno** [comment]
I noticed the values for context menu were old. I updated them. It should work now.

**Pull request merged**